public/wcif-extensions/ActivityConfig.json

```diff
@@ -28,6 +28,10 @@
     "assignJudges": {
       "description": "A flag indicating whether judges should be assigned to the given activity.",
       "type": "boolean"
+    },
+    "featuredCompetitorIds": {
```

Comment:
Probably better to be explicit and say `featuredCompetitorWcaIds`.

Reply:
I switched to featuredCompetitorWcaUserIds to make it clear this is not WCA IDs.

Reply:
Ah perfect, that's what I meant actually :D